asab/library/service.py

```diff
@@ -12,69 +12,94 @@
 #
 
 
-asab.Config.add_defaults(
```

Comment:
@ateska Can I know why Defaults was moved from service to init?

Reply:
No specific reason, honestly.
`__init__.py` is more typical/intuitive location for default configs, I guess.